pkgs/sops-install-secrets/main.go

```diff
@@ -116,13 +117,25 @@ func readManifest(path string) (*manifest, error) {
 	return &m, nil
 }
 
+func linksAreEqual(linkTarget, targetFile string, info os.FileInfo, secret *secret) bool {
+	validUG := true;
+	if stat, ok := info.Sys().(*syscall.Stat_t); ok {
```

Comment:
I think this should panic in the `!ok` case. This way if we ever target a new operating system this error does not get hidden.